{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--cc3bf823-1ee7-4a47-b060-93ef4bc95522",
    "created": "2024-08-13T16:15:11.310105Z",
    "modified": "2024-08-13T16:15:11.31011Z",
    "name": "No Title Available",
    "description": "An issue was discovered in proxy.php in pydio-core in Pydio through 8.2.2. Through an unauthenticated request, it possible to evaluate malicious PHP code by placing it on the fourth line of a .php file, as demonstrated by a PoC.php created by the guest account, with execution via a proxy.php?hash=../../../../../var/lib/pydio/data/personal/guest/PoC.php request. This is related to plugins/action.share/src/Store/ShareStore.php.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2019-9642"
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://github.com/pydio/pydio-core/commits/develop/core/src/proxy.php"
        },
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "https://pydio.com/en/community/releases/pydio-core/pydio-core-pydio-enterprise-823-security-release"
        }
    ]
}